templates/Web/_composition/Flask/Page.Flask.List.AddRoutes.WithCosmos.SQL/server/server_postaction.py

```diff
@@ -12,37 +12,37 @@
 app = Flask(__name__)
 
 //{[{
-sqlDatabaseObj = SQLObj()
+sql_database_obj = SQLObj()
 
 # List Endpoints
 @app.route(CONSTANTS['ENDPOINT']['LIST'])
-def getList():
+def get_list():
     queryStr = {
```

Comment:
should queryStr be underscore separated too?

Reply:
Missed that one. Thanks for pointing it out :)